README.md

```diff
@@ -93,101 +94,90 @@ Retrieves a list of orders given query parameters. Default is all open orders.
+            "takerTokenAmount": "20000000000000000",
+            "makerFee": "100000000000000",
+            "takerFee": "200000000000000",
             "expirationUnixTimestampSec": "42",
+            "salt": "256",
             "ecSignature": {
                 "v": 27,
                 "r": "0x61a3ed31b43c8780e905a260a35faefcc527be7516aa11c0256729b5b351bc33",
                 "s": "0x40349190569279751135161d22529dc25add4f6069af05be04cacbda2ace2254"
             }
-        },
-        "state": "OPEN",
-        "pending": {
-            "fillAmount": "50000000000000000",
-            "cancelAmount": "50000000000000000"
-        },
-        "remainingTakerTokenAmount": "1000000000000000000"
-    }
+        }
+    },
     ...
 ]
 ```
 
-- `state` - the state of the order. The order is `OPEN` until completely filled/canceled.
 
-### GET /v0/order/[orderHash]
+### GET /v0/orders/[orderHash]
 
 Retrieves a specific order by orderHash.
 
 #### Response
 
-[See response schema](https://github.com/0xProject/json-schemas/blob/master/schemas/relayer_api_order_response_schema.ts#L7)
+[See response schema](https://github.com/0xProject/json-schemas/blob/master/schemas/relayer_api_orders_response_schema.ts#L7)
 
 
 ```
 {
     "signedOrder": {
+        "exchangeContractAddress": "0x12459C951127e0c374FF9105DdA097662A027093",
         "maker": "0x9e56625509c2f60af937f23b7b532600390e8c8b",
         "taker": "0xa2b31dacf30a9c50ca473337c01d8a201ae33e32",
-        "makerFee": "100000000000000",
-        "takerFee": "200000000000000",
-        "makerTokenAmount": "10000000000000000",
-        "takerTokenAmount": "20000000000000000",
         "makerTokenAddress": "0x323b5d4c32345ced77393b3530b1eed0f346429d",
         "takerTokenAddress": "0xef7fff64389b814a946f3e92105513705ca6b990",
-        "salt": "256",
         "feeRecipient": "0xB046140686d052ffF581f63f8136CcE132e857dA",
-        "exchangeContractAddress": "0x12459C951127e0c374FF9105DdA097662A027093",
+        "makerTokenAmount": "10000000000000000",
+        "takerTokenAmount": "20000000000000000",
+        "makerFee": "100000000000000",
+        "takerFee": "200000000000000",
         "expirationUnixTimestampSec": "42",
+        "salt": "256",
         "ecSignature": {
             "v": 27,
             "r": "0x61a3ed31b43c8780e905a260a35faefcc527be7516aa11c0256729b5b351bc33",
             "s": "0x40349190569279751135161d22529dc25add4f6069af05be04cacbda2ace2254"
         }
-    },
-    "state": "OPEN",
-    "pending": {
-        "fillAmount": "50000000000000000",
-        "cancelAmount": "50000000000000000"
-    },
-    "remainingTakerTokenAmount": "10000000000000000"
+    }
 }
 ```
 
 Returns HTTP 404 if no order with specified orderHash was found.
 
 ### POST /v0/fees
 
-Returns the only acceptable fees (in ZRX) for the specified order parameters
+Given an unsigned order with no fees, returns the required `feeRecipient`, `makerFee`, and `takerFee` of that order.
```

Comment:
"with no fees..." sounds like I just omit `makerFee` and `takerFee`. What about: "without the fee-related properties..."